scripts/e2e-tests.sh

```diff
@@ -82,7 +82,7 @@ function go_test_e2e() {
 # Download the k8s binaries required by kubetest.
 # Parameters: $1 - GCP project that will host the test cluster.
 function download_k8s() {
-  local version=${E2E_CLUSTER_VERSION}
+  local version=${E2E_DEFAULT_CLUSTER_VERSION}
```

Comment:
I think we can remove E2E_DEFAULT_CLUSTER_VERSION, as it's not adding value here. Then, in line 113, readonly E2E_CLUSTER_VERSION, as it contains now the real version number.